example/package.json

```diff
@@ -4,13 +4,14 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "react-scripts start",
+    "start": "env-cmd -f .env.local react-scripts start",
```

Comment:
Good, maybe let's add in the readme that one needs to create this file. Should this file go in the root of the project?